libs/designsystem/icon/src/icon.component.ts

```diff
@@ -18,6 +17,8 @@ export enum IconSize {
   LG = 'lg',
 }
 
+const CUSTOME_NAME_DEPRECATION_WARNING = 'customName input property depcrecation warning';
```

Comment:
```suggestion
const CUSTOM_NAME_DEPRECATION_WARNING = 'customName input property depcrecation warning';
```

I also think this warning would benefit from being a bit more descriptive. 
Our other deprecation warnings have the following format:
`Deprecation warning: selectedSlide will be removed in Kirby v10. Use slideChange instead.`
or 
`Deprecation warning: The "kirby-table" support for using the input property "fixedLayout" will be removed in the next major version. We recommend to use the global CSS class "layout-fixed" in combination with the "kirby-table" class'`

Reply:
I think we can be specific about remnoving it in v10 here.